pkg/api/v1/deep_copy_generated.go

```diff
@@ -1527,6 +1527,24 @@ func deepCopy_v1_RollingDeploymentStrategyParams(in deployapiv1.RollingDeploymen
 	} else {
 		out.TimeoutSeconds = nil
 	}
+	if in.MaxUnavailable != nil {
```

Comment:
Yeah... if I truncate that file and re-run `hack/update-generated-deep-copies.sh`, I get no diffs.